### PR TITLE
add support for global forwarding rule

### PR DESCRIPTION
**Pull request by ericyz**

### CODE REVIEW

**Comment by ericyz** [comment]
Hi @melinath , this PR is ready for review

File: converters/google/mappers.go [hunk 1]
```diff
@@ -50,17 +50,19 @@ type mapper struct {
 func mappers() map[string][]mapper {
 	return map[string][]mapper{
 		// TODO: Use a generated mapping once it lands in the conversion library.
-		"google_compute_firewall":      {{convert: converter.GetComputeFirewallCaiObject}},
-		"google_compute_disk":          {{convert: converter.GetComputeDiskCaiObject}},
-		"google_compute_instance":      {{convert: converter.GetComputeInstanceCaiObject}},
+		"google_compute_firewall":               {{convert: converter.GetComputeFirewallCaiObject}},
+		"google_compute_disk":                   {{convert: converter.GetComputeDiskCaiObject}},
+		"google_compute_forwarding_rule":        {{convert: converter.GetComputeForwardingRuleCaiObject}},
 		"google_storage_bucket":        {{convert: converter.GetStorageBucketCaiObject}},
```
Comment (melinath) [inline converters/google/mappers.go:56]: It looks like this line snuck in here & is causing test compilation to fail.
```suggestion
```

**Review by melinath — CHANGES_REQUESTED** [review]

**Review by melinath — CHANGES_REQUESTED** [review]
Tests are failing - it looks like the underlying issue may be:

```
ERROR: logging before flag.Parse: I0222 07:31:27.689205    7315 convert.go:187] unsupported resource: google_compute_global_forwarding_rule
```

If looks like the `mappers` contains `google_compute_forwarding_rule` instead of  `google_compute_global_forwarding_rule` - should it include `_global_`?

**Review by melinath — APPROVED** [review]
LGTM

**Pull request merged**